src/main/kotlin/no/nav/syfo/application/ForesporselService.kt

```diff
@@ -1,12 +1,21 @@
 package no.nav.syfo.application
 
 import no.nav.syfo.domain.Foresporsel
+import no.nav.syfo.domain.Personident
 
-class ForesporselService(private val varselProducer: IVarselProducer) {
-    fun sendToNarmesteleder(foresporsel: Foresporsel): Result<Foresporsel> {
+class ForesporselService(
+    private val varselProducer: IVarselProducer,
+    private val repository: IForesporselRepository,
+) {
+    fun storeAndSendToNarmesteleder(foresporsel: Foresporsel): Result<Foresporsel> {
```

Comment:
Å lagre (`store`) er kanskje en teknisk detalj? Er det servicen egentlig skal gjøre her å sende denne forespørselen til nærmeste leder? 😊
```suggestion
    fun sendToNarmesteleder(foresporsel: Foresporsel): Result<Foresporsel> {
```

Reply:
Ja, kan godt endre tilbake igjen. Evnt kunne det ha vært tjenesten vært `createForesporsel `som vil samsvare bedre med hvordan vi har gjort det i noen av de andre app'ene.